README.md

```diff
@@ -190,6 +190,14 @@ Usage: very_good test [arguments]
 Run "very_good help" to see global options.
 ```
 
+#### Tests without pub install
+
+Unlike `flutter test`, `very_good test` will always run your tests without installing the projects dependencies (i.e. `--no-pub` flag).
+
+This is part of the test optimization provided by the cli, since usually the only time where dependencies installation would be required is on a fresh clonned repository, and a developer would normally be running tests multiple times on their local repository, by not running `pub get` we gain a few seconds since no network call to pub is made.
```

Comment:
```suggestion
This is an optimization done by the cli because dependency installation is usually run once after cloning the repository. Conversely, running tests locally is usually done many times and it's often unnecessary to re-install dependencies prior to each test run.
```